src/support/analyzer_annotations.h

```diff
@@ -10,6 +10,7 @@
 #define JL_ALWAYS_LEAFTYPE __attribute__((annotate("julia_always_leaftype")))
 #define JL_ROOTS_TEMPORARILY __attribute__((annotate("julia_temporarily_roots")))
 #define JL_REQUIRE_ROOTED_SLOT __attribute__((annotate("julia_require_rooted_slot")))
+#define JL_ANALYZER_HIDDEN_BODY(x) {}
```

Comment:
Document these too? Is this just for functions that are too complex?

Reply:
Since it's only used twice, maybe just use `ifdef __clang_analyzer__`? Autoformat tools hate it when you use macros like this.

Reply:
You're right. Will undo, though it is uglier.